planetiler-core/src/test/java/com/onthegomap/planetiler/collection/FeatureGroupTest.java

```diff
@@ -293,6 +296,84 @@ public List<VectorTile.Feature> postProcessLayerFeatures(String layer, int zoom,
       )), getFeatures());
   }
 
+  @Test
+  void testHilbertOrdering() {
+    features = new FeatureGroup(sorter, TileOrder.HILBERT, new Profile.NullProfile() {}, Stats.inMemory());
+    featureWriter = features.writerForThread();
+
+    // TMS tile IDs at zoom level 1:
+    // 2 4
+    // 1 3
```

Comment:
```suggestion
    // 1 2
    // 4 3
```

Reply:
Don't all X=0 come before X=1? unless I am misreading the TileCoord code...

Reply:
For hilbert I think it should be:

```java
    // Hilbert tile IDs at zoom level 1:
    // 1 4
    // 2 3
```